index.html

```diff
@@ -13,16 +10,72 @@
   </head>
   <body>
 
+<!--        header            -->
     <header>
-      
-    </header>
+      <section class="left-nav-container">
+
+        <div class="nav-choice-container">
+          <img class="icon" src="./assets/GPS.svg" alt="gps icon">
+          <p class="nav-link">navlink</p>
+        </div>
 
+        <div class="nav-choice-container">
+          <img class="icon" src="./assets/GPS.svg" alt="gps icon">
+          <p class="nav-link">navlink</p>
+        </div>
+
+        <div class="nav-choice-container">
+          <img class="icon" src="./assets/GPS.svg" alt="gps icon">
+          <p class="nav-link">navlink</p>
+        </div>
+
+      </section>
+      <section class="right-nav-container">
+        <div class="user-container">
+          <img class="icon-user" src="./assets/GPS.svg" alt="gps icon">
+          <p class="user">Username</p>
+        </div>
+        <img class="logo" src="./assets/GPS.svg" alt="placeholder">
+      </section>
+    </header>
+<!--          mainsection            -->
```

Comment:
There are opinions about leaving comments in code, and personally my opinion the more comments the better. There's some saying that goes like, "you will spend more time reading code than writing it". If I were to nitpick I could say that this comment and the one with the header are redundant because you are using semantic html by having header and main elements in your file. Try to find that middle ground of does this comment add more info or is it restating. But do leave comments